warehouse/attestations/services.py

```diff
@@ -190,6 +193,17 @@ def parse_attestations(
                     f"Attestation with unsupported predicate type: {predicate_type}",
                 )
 
+            if predicate_type in seen_predicate_types:
+                self.metrics.increment(
+                    "warehouse.upload.attestations.failed_duplicate_predicate_type"
+                )
+                raise AttestationUploadError(
+                    f"Multiple attestations for the same file with the same predicate "
+                    f"type: {predicate_type}",
```

Comment:
```suggestion
                    f"Multiple attestations for the same file with the same predicate "
                    f"type: {predicate_type!r} is not supported",
```

Reply:
Here the error is not that `predicate_type` is not supported, but rather that multiple attestations with the same predicate type were uploaded.

Reply:
Yes, I think that's still conveyed by the error message?

Reply:
But the suggestion says to add "{predicate_type} is not supported"

Reply:
I think you're mis-reading the diff here.

Reply:
Got it. I reworded it a bit to:
```python
f"Multiple attestations for the same file with the same "
f"predicate type ({predicate_type.value}) are not supported",
```

Also check the last commit for a minor change on how we handle the enum value for `predicate_type`